### PR TITLE
docs: fix socketio's browser demo

### PR DESCRIPTION
**Pull request by x-cold**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

https://github.com/eggjs/egg/issues/2601

### CODE REVIEW

File: docs/source/en/tutorials/socketio.md [hunk 1]
```diff
@@ -331,13 +331,14 @@ The UI-related content is not rewritten. It can be called via window.socket
 // browser
 const log = console.log;
 
-window.onload = function() {
+window.onload = function () {
   // init
   const socket = io('/', {
-    // Actual use can pass parameters here
+
+    // 实际使用中可以在这里传递参数
```
Comment (vincent178) [inline docs/source/en/tutorials/socketio.md]: 在英文文档里面写中文注释不好吧

File: docs/source/en/tutorials/socketio.md [hunk 2]
```diff
@@ -331,13 +331,14 @@ The UI-related content is not rewritten. It can be called via window.socket
 // browser
 const log = console.log;
 
-window.onload = function() {
+window.onload = function () {
```
Comment (vincent178) [inline docs/source/en/tutorials/socketio.md]: 这种格式化的改动应该去掉

Comment (x-cold) [reply docs/source/en/tutorials/socketio.md]: Done

File: docs/source/zh-cn/tutorials/socketio.md [hunk 3]
```diff
@@ -342,14 +342,14 @@ UI 相关的内容不重复写了，通过 window.socket 调用即可
...
-window.onload = function () {
+window.onload = function() {
   // init
   const socket = io('/', {
 
     // 实际使用中可以在这里传递参数
     query: {
       room: 'demo',
-      userId: `client_${Math.random()}`,
+      userId: `client_${Math.random()}`
```
Comment (dead-horse) [inline docs/source/zh-cn/tutorials/socketio.md]: 保留这个逗号

**Pull request merged**